added domain registry (#60)
argoapp-live/argo-docs: versioned_docs/version-3.0.0-beta/domain-registry/secure-on.mdx

---

id: secure-on

> title: Adding your domain to secureon

Reviewer: ```suggestion
title: Adding your domain to Secureon
```

sidebar_label: Adding domain
slug: /secure-on

---

## secureon

Secureon is a frontend validation tool powered by Spheron
At Secureon, we are building extensions to validate any Dapp. Due to the lack of security in accessing Dapps that handle millions of dollars of crypto, we are building validation tools for the community to validate and access only secured frontends.

We ensure users are only accession dapps deployed via Spheron and maintained or curated by verified entities or governance.
This extension validates the user interface by making sure the content serving on the UI is the same as deployed by the team.

With the help of an extension, users don't have to trust any email-based links, worry about domain hijacking, or fear a centralized domain getting down. Now, users will always have a decentralized version of the web apps they can open.

## How to add domain to secureon

-   <strong>

>         Go to your project dashboard and click on domain as shown below

Reviewer: ```suggestion
        Go to your project dashboard and click on the domain tab as shown below
```

    </strong>

<img src={require("@site/static/img/3.0.0/pdb.png").default} />

-   <strong>

>         From the sidebar on the left-hand side panel,navigate to your subdomain

Reviewer: ```suggestion
        From the sidebar on the left-hand side panel, navigate to your subdomain
```

    </strong>

<img src={require("@site/static/img/3.0.0/prd.png").default} />

-   <strong>

>         Click on update on secureon for any of your sub-domains and choose your preferred registry

Reviewer: ```suggestion
        Click on update on Secureon for any of your subdomains and choose your preferred registry
```

    </strong>

<img src={require("@site/static/img/3.0.0/aso.png").default} />

-   <strong>

>         After selecting your preferred registry,click on add and confirm notification prompt from metamask/your choiced wallet.

Reviewer: ```suggestion
        After selecting your preferred registry, click on add and confirm notification prompt from metamask / your chosen wallet.
```

    </strong>

<img src={require("@site/static/img/3.0.0/msp.png").default} />

-   <strong>

>         After confirming the prompt,you will be redirected to a page to show your domain has been added successfully.

Reviewer: ```suggestion
        After confirming the prompt, you will be redirected to a page to show your successfully added domains.
```
